### DISCUSSION TITLE
想多学一点Hugo和这个主题的知识

### DISCUSSION THREAD

**Discussion by HonorKnight8** [HEIGE-PCloud/DoIt#109] [General]
现在虽然会简单使用一下，但很多功能其实还是一头雾水，请问要如何进一步学习呢

**Comment by wu-yue-yu** [comment]
请看官方中文文档，非常详细，我也用的这个主题
[文档链接](https://hugodoit.pages.dev/zh-cn/theme-documentation-basics/#site-configuration)
